mojo/services/keyboard/interfaces/BUILD.gn

```diff
@@ -9,4 +9,10 @@ mojom("interfaces") {
   sources = [
     "keyboard.mojom",
   ]
+
+  import_dirs = [ get_path_info("../../", "abspath") ]
+
+  deps = [
+    "../../native_viewport/interfaces",
```

Comment:
This is deliberate so the mojom definitions can be used in other repos that want to check mojo/services out under a different root (such as //third_party/mojo/services/...). Chinmay is right that most paths are absolute, not relative, but this tree is an exception.